Content.Shared/Explosion/Components/OnUseTimerTriggerComponent.cs

```diff
@@ -1,57 +1,49 @@
 using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
 
-namespace Content.Server.Explosion.Components
+namespace Content.Shared.Explosion.Components
 {
-    [RegisterComponent]
+    [RegisterComponent, NetworkedComponent]
     public sealed partial class OnUseTimerTriggerComponent : Component
     {
-        [DataField("delay")]
-        public float Delay = 1f;
+        [DataField("delay")] public float Delay = 1f;
```

Comment:
Why change this? In most files it's on separate lines.

Reply:
> Why change this? In most files it's on separate lines.

Old formatting mistake.